guide/17-geocoding-guides/part1_what_is_geocoding.ipynb

```diff
@@ -0,0 +1,1334 @@
+{
```

Comment:
<p>Note here, you can either create the <code>GIS</code> with username and password or with the <code>api_key</code> parameter, e.g. <code>gis = GIS(api_key=&lt;your api key&gt;)</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>